src/main/java/seedu/address/logic/commands/FindScheduleCommand.java

```diff
@@ -0,0 +1,132 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DASH_DAY_SHIFT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DASH_TIME;
+
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+
+import javafx.collections.ObservableList;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.predicates.PersonIsWorkingPredicate;
+
+
+/**
+ * Class representing the view schedule command
+ * which views the schedule by Person.
+ */
+public class FindScheduleCommand extends Command {
+    public static final String DEFAULT_MESSAGE = "Staff working at that shift: \n";
+    public static final String COMMAND_WORD = "findSchedule";
+    public static final String HELP_MESSAGE = COMMAND_WORD + ": find the staff working at the specified shift\n\n"
+            + "Parameters:\n"
+            + PREFIX_DASH_DAY_SHIFT + " [DAY]-[SLOT NUMBER]\n"
```

Comment:
Hmm sorry not sure about the square brackets cause they're not optional! HAHA sorry I didn't see it just now 😭